cclib/io/moldenwriter.py

```diff
@@ -244,6 +244,8 @@ def reformat(self):
 
             # Check for [Title] section.
             if '[title]' in line.lower():
+                # skip the title
+                line = next(filelines)
```

Comment:
Can you use `self.skip_lines` here?

Reply:
So I could for the line that actually says [title], but we need to skip the line after it which is the real title and that could be anything. I was looking at [this](https://github.com/cclib/cclib/blob/5605bdd98085a7e41dee4965bf0573aeefe89b2e/cclib/parser/logfileparser.py) and it looks like I have to give it some pattern to skip right?

Reply:
We could stand to update the docs here, because it also confused me for the longest time. Here is what it actually does. Let's say I write
```python
self.skip_lines(inputfile, ["d", "b", "Excitation energy in a.u."])
line = next(inputfile)
```
where `d` is for a line that contains only dashes (and possibly spaces, but excluding the newline) and `b` is for a "blank" line that contains zero or more spaces (excluding the newline). For dashes, stars, equals signs, or a blank line, we do a check to make sure that the line is exactly that. Even if it is not, the line is still skipped after the warning is printed. For anything else, like `"Excitation energy in a.u."` or `"authors"` or `"State   Energy Wavelength"`, it's a comment to the person reading the code about what that line contains, and no check is performed on the line before it's skipped. TL;DR it's just a comment meant for us, not a pattern to be matched.

Reply:
This is correct. We should update the docs, and maybe make the the syntax more explicit.

Reply:
Created #765 for this.

Reply:
Ok I think I understand how this work, but `Molden_reformatter` isn't of type `cclib.parser.logfileparser.Logfile` or any derived class. I'm still confused about how to use skip_lines outside of such a class?

Reply:
Ah, you're right. Scratch that then.